ambry-clustermap/src/main/java/com.github.ambry.clustermap/AmbryPartition.java

```diff
@@ -28,6 +29,7 @@
   private final Long id;
   private final ClusterManagerCallback clusterManagerCallback;
   private volatile PartitionState state;
+  private final AtomicLong lastUpdatedSealedStateChangeCounter = new AtomicLong(-1);
```

Comment:
Just initializing to a value less than the initial value in the cluster manager to ensure that the first time it actually gets the state by querying the replica states.